packages/liquidator/test/Liquidator.js

```diff
@@ -1732,11 +1742,21 @@ contract("Liquidator.js", function (accounts) {
             // Seed the market. This sets up the initial price to be 1/1 reserve to collateral token. As the collateral
             // token is Dai this starts off the uniswap market at 1 reserve/collateral. Note the amount of collateral
             // is scaled according to the collateral decimals.
-            await reserveToken.mint(pairAddress, toBN(toWei("1000")).muln(10000000), { from: contractCreator });
```

Comment:
rather than minting to the exchange, we use the `addLiquidity` method to enable us to pull liquidity later on to validate slippage behavour.